src/actions/create.js

```diff
@@ -18,7 +20,7 @@ import thunkAction from './_thunkAction';
  * @param item holds object that you want to pass to api
  * @returns {function}
  */
-export function create(config, schema, item = null) {
+export function create(config, schema, item = null, params = {}, options = {}) {
```

Comment:
signature should be `create(schema, params = {}, options = {})` and schema should be resolved by name or by object same as in `find`